src/protocols/ec_udp.c

```diff
@@ -111,7 +111,7 @@ FUNC_DECODER(decode_udp)
           *
           * if the source is the ettercap host, don't display the message 
           */
-         if (ip_addr_is_ours(&PACKET->L3.src) == E_FOUND || ip_addr_is_ours(&PACKET->L3.src) == E_BRIDGE)
+         if (ip_addr_is_ours(&PACKET->L3.src) != E_FOUND && ip_addr_is_ours(&PACKET->L3.src) != E_BRIDGE)
             return NULL;
```

Comment:
@koeppea do you have any clue about this commit I did?
https://github.com/LocutusOfBorg/ettercap/commit/79ceb97c3fee6d2e5924902507a355d4a7597d68
I don't remember the rationale for adding the second comparison to the if...